hw/ip/kmac/dv/kmac_sim.core

```diff
@@ -0,0 +1,26 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:kmac_sim:0.1"
+description: "KMAC DV sim target"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:kmac
+
+  files_dv:
+    depend:
+      - lowrisc:dv:kmac_test
+      - lowrisc:dv:kmac_sva
+    files:
+      - tb.sv
+    file_type: systemVerilogSource
+
+targets:
+  sim:
```

Comment:
BTW, please also copy some update from this [PR](https://github.com/lowRISC/opentitan/pull/4023/files) and add kmac in top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson in order to enable DV lint.

Reply:
It's also fine if you prefer to do it in another PR.

Reply:
Will do, I planned to add that in a separate PR but I can definitely do that here. thanks for the suggestion!